php/WP_CLI/Dispatcher/Subcommand.php

```diff
@@ -325,7 +326,17 @@ private function validate_args( $args, $assoc_args, $extra_args ) {
 
 		if ( $this->name != 'help' ) {
 			foreach ( $validator->unknown_assoc( $assoc_args ) as $key ) {
-				$errors['fatal'][] = "unknown --$key parameter";
+				$suggestion = Utils\get_suggestion(
+					$key,
+					$this->get_parameters( $synopsis_spec ),
+					$threshold = 2
```

Comment:
It's a way of better documenting the code, instead of only having a raw scalar value in there. Using such an assignment makes it obvious what the purpose of the `2` is.